src/backend/joanie/core/api/admin.py

```diff
@@ -221,6 +221,8 @@ def create(self, request, *args, **kwargs):
         data = request.data
         serializer = self.get_serializer(data=data)
         data["product"] = kwargs.get("product_id")
+        if not data.get("course_runs", None):
```

Comment:
Can you add a comment to explain that you must fix the incoming data before going through validation?

Reply:
Can you make the same modification for partial_update?